Line/android/LINELoginDemo/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.google.firebase.linelogindemo">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application
+        android:name=".LineLoginDemoApp"
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
```

Comment:
Fixed in this [commit](https://github.com/firebase/custom-auth-samples/pull/1/commits/fd56eda6713e7fd016d5c8ad820c78d2ffda97bf)